tests/integration/explain/default/create_test.go

```diff
@@ -80,32 +79,19 @@ func TestExplainMutationCreateSimpleDoesNotCreateDocGivenDuplicate(t *testing.T)
 			},
 		},
 
-		Results: []dataMap{
+		ExpectedTargets: []explainUtils.PlanNodeTargetCase{
```

Comment:
question: It seams as if a lot of the changes to the tests, including this one, are testing less of the output. Am I missing something or is this the intent?

Reply:
It's now testing the pattern (ordering of the nodes), and only the content of the nodes that we care about for each test separately. For this specific test for example we only care about the `createNode`'s attributes (point 2 from the below link).

More info can be found here:
https://github.com/sourcenetwork/defradb/pull/949#issuecomment-1327719247

Reply:
Thanks for referencing that comment! It makes much more sense to me now :)